locale/en.po

```diff
@@ -1726,6 +1726,73 @@ msgstr ""
 "version.\n"
 "Please load the save manually through the menu."
 
+#: ../src/gui_common/TweakedColourPicker.cs:211
+msgid "COLOR_PICKER_PICK_COLOR"
+msgstr "Pick a color from window"
+
+#: ../src/gui_common/TweakedColourPicker.cs:212
+msgid "COLOR_PICKER_ADD_PRESET"
+msgstr "Add current color as a preset"
+
+#: ../src/gui_common/TweakedColourPicker.cs:213
+msgid "COLOR_PICKER_HSV_BUTTON_TOOLTIP"
+msgstr "Switch HSV (Hue, Saturation, Value) mode on or off.\n"
+"Can't be truned on when in Raw mode."
```

Comment:
If "Raw" is capitalized, shouldn't maybe mode be as well? Usually if you have something like that the full name is like "Raw Mode" so the second part of the name is also capitalized.
But maybe "raw" shouldn't be capitalized either? I don't really see the need why it would need to be capitalized, "raw" input is a pretty common term, I think, so it isn't a name that needs to be capitalized.